### PR TITLE
Gbunce 911 solutions page

### PR DESCRIPTION
**Pull request by steveoh**
i wanna seeee

### CODE REVIEW

**Review by steveoh — COMMENTED** [review]
It would have been more ideal to fix these links in a new branch and leave the old website content alone

File: data/index.html [hunk 1]
```diff
@@ -276,7 +276,7 @@ <h2>Choosing a Data Source</h2>
     <div class="grid__col section alternate">
```
[the remark this answers is not in the document]
Comment (gregbunce) [reply data/index.html]: done

File: about/index.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ tags: []
 categories: []
```
[the remark this answers is not in the document]
Comment (gregbunce) [reply about/index.md:8]: done

**Comment by steveoh** [comment]
@gregbunce i had to rebase to fix some conflicts and get you up to speed on main. `git pull --rebase` before you continue working locally please.

**Comment by gregbunce** [comment]
Will do

**Comment by gregbunce** [comment]
> @gregbunce i had to rebase to fix some conflicts and get you up to speed on main. `git pull --rebase` before you continue working locally please.

done

**Comment by gregbunce** [comment]
@steveoh let's close this out!  i'm ready for a final review.

**Comment by steveoh** [comment]
> @steveoh let's close this out! i'm ready for a final review.

image

**Comment by gregbunce** [comment]
i don't see that button, maybe b/c i didn't put it the PR (?).

**Comment by gregbunce** [comment]
btw: i like the rick voice. ha

**Review by steveoh — COMMENTED** [review]
I like it. It's maybe a little heavy on what we do vs what solutions we offer ems folks but it is in there if you look for it. let's get this in and if you find any motivation to lean a bit more towards solutions we can rework it a bit. 

So I understand, do we work with PSAP's and do other ETL type work for them that earn us some money? Maybe the hooks for things that they should  be using from us, besides data, is missing and what we can be contracted to help them with is maybe missing?

**Pull request merged**